internal/contour/route_test.go

```diff
@@ -237,7 +237,7 @@ func TestRouteVisit(t *testing.T) {
 					Name: "ingress_http",
 					VirtualHosts: []route.VirtualHost{{
 						Name:    "www.example.com",
-						Domains: []string{"www.example.com", "www.example.com:80"},
+						Domains: []string{"www.example.com", "www.example.com:*"},
```

Comment:
suggestion: the value of `Domains` can be determined fully from its input, try making a small helper to dry this up.
```go
func domains(hostname string) []string {
 return []string{hostname, hostname+":*"}
}
```